Python/stringSubsetsRecursive.py

```diff
@@ -0,0 +1,21 @@
+#author Matteo Messmer https://github.com/matteomessmer
+
+#to simplify the function
+def strSubsets(original):
+    return strSubsetsRecursive(original, '', [''])
+
+def strSubsetsRecursive(original, accumulator, subsets):
+    if len(original) > 0:
+        currentChar = original[0]
+
+        #go to the next char and add it to the accumulator
+        strSubsetsRecursive(original[1:], accumulator + currentChar, subsets)
```

Comment:
Unnecessarily slow code, please don't use string slicing in python, you can use a current index value to be used

Reply:
It's fine we can accept slicing